src/plugins/src/datanode.rs

```diff
@@ -16,10 +16,16 @@ use common_base::Plugins;
 use datanode::config::DatanodeOptions;
 use datanode::error::Result;
 
-pub async fn setup_datanode_plugins(_opts: &mut DatanodeOptions) -> Result<Plugins> {
-    Ok(Plugins::new())
+#[allow(unused_variables)]
+#[allow(unused_mut)]
+pub async fn setup_datanode_plugins(
+    plugins: &mut Plugins,
+    dn_opts: &DatanodeOptions,
```

Comment:
Can we use `_plugins` to avoid `#[allow(unused_variables)]`, or it's a deliberate choice?

Reply:
Actually, I did it on purpose because we don't change the signature when we override the plugins crate.